### ISSUE TITLE
Need your help with longer lk4 and marlin

### ISSUE THREAD

**Issue by danielgalizi** [DeviousPenguin/test#1]
Hello. Sorry, but it was the only way to send.you a message.
I Need some help with klipper and lk4 pro. Can you help me? Thanks. DG

**Comment by DeviousPenguin** [comment]
Hi, no problem, I've just checked notifications, so sorry for the late reply.

What is the problem you are having with klipper + the LK4 pro?

**Comment by danielgalizi** [comment]
Hello!! thanks for the reply!!! and sorry for my late reply :-D
I cannot find the configuration file for LK4 PRO. tried to configure myself, but no success... could you send me the configs you used to setup yours? 
 if you want me to contact you outside this chat, just let me know.
again, thank you very much for the reply.

**Comment by DeviousPenguin** [comment]
I pretty much used the Klipper config for an Ender 3 but changed the pins to match the LK4 pro board. I'll upload the configs to this 'test' repo in a bit.

My LK4 has been modified however, I'm no longer using the Z endstop and now using a BL Touch.

I'm also using different pins for the fans, as I shorted and killed one of the fan mosfets, and also added UART wires to the 2208 stepper drivers. What I'm trying to say it you can't just download and use this config, they'll need some tweaking in order to work with a stock LK4 Pro, so you will need to use my config as a template in order to have a working printer.

EDIT: `printer.cfg` added :)

**Comment by danielgalizi** [comment]
Hello!! thanks!! i will take a look. I also have bltouch.
do you note any difference in printing with klipper?
thank you very much!!

**Comment by DeviousPenguin** [comment]
No problem, I didn't print a huge amount of things when using Marlin so it's hard to say what the differences are compared to Klipper. I stopped using Marlin after I was unable to get the BL Touch to work with it.

I find the main nice thing about Klipper is that you can easily change most printer settings without having to reflash the MCU each end every time like you do with Marlin.

**Comment by danielgalizi** [comment]
yes, you are right....and because of the low memory, always with errors when compiling...
i was able to configure the bltouch with marlin, but sometimes looks like it is not working well.... even after to level the bed, and run a bltouch check, the first layers are very bad at one side... ill give it a try with klipper. I keep you posted. if you want to keep in touch, daniel.galizi at gmail dot com.
thanks man.
DG

**Comment by DeviousPenguin** [comment]
Just a quick heads up, the Longer LK4 Pro has been officially added to Klipper recently:

https://github.com/KevinOConnor/klipper/blob/master/config/printer-longer-lk4-pro-2019.cfg

This config should work far better than my settings as they are probably using the original pins for the board

**Comment by Sinclair5** [comment]
hi sorry to disturb but I'm having issues getting the filament runout sensor to work.   i have used your config and the one from klipper but it either says the pin isn't available or  its been used too much.  what do I do?

**Comment by Knustl** [comment]
Pode me ajudar na configuração da minha Longer LK4 Pro no Klipper ( Fluidd )?
Obs: Já está instalada e funcionando. Mas não sai uma impressão boa, sai muito ruim. Uso o Orca Slicer e não estou conseguindo as configurações para fatiar.
Não possuo BLTouch.
